source/release-notes/4-2-0.rst

```diff
@@ -0,0 +1,49 @@
+.. index:: CrafterCMS 4.2.0 Release Notes
+
+----------------
+CrafterCMS 4.2.0
+----------------
+
+Released *TBD*
+
+.. important::
+
+    Please see :ref:`Breaking changes <breaking-changes-4-2-x>` for breaking changes and other
+    things to consider before upgrading
+
+    For instructions on how to update your CrafterCMS install to 4.2.x,
+    please see :ref:`upgrading-craftercms`
+
+^^^^^^^^^^^^
+New Features
+^^^^^^^^^^^^
+
+^^^^^^^^^^^^
+Enhancements
+^^^^^^^^^^^^
+
+^^^^^^^^^
+Bug Fixes
+^^^^^^^^^
+
+^^^^^^^^^^^^^^^^^^
+Dependency Updates
+^^^^^^^^^^^^^^^^^^
+* Noteworthy dependency updates:
+
+  - Spring Framework to version 6
+  - Spring Boot to version 3
+  - Freemarker to version 2.3.33
+  - MariaDb4J to version ``3.1.0.1`` and MariaDB to ``11.3.2``
+  - HTTP Proxy to version 2.0
+  - Tuckey to version 5.1.3
```

Comment:
Is this the right dependency name? I thought it was `UrlRewriteFilter` or similar.